### ISSUE TITLE
Need for Normalization of Pixel Data can be mentioned in Basic Image Classification Tutorial

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorflow#47526]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/tutorials/keras/classification#preprocess_the_data

## Description of issue (what needs changing): 
The Tutorial states 

> Scale these values to a range of 0 to 1 before feeding them to the neural network model.

It would be clear to the beginners if the reason for **`Normalization`** has been specified because there has been questions out of confusion. Example [Question 1](https://datascience.stackexchange.com/questions/39929/why-normalize-when-all-features-are-on-the-same-scale), [Question 2](https://stackoverflow.com/questions/20486700/why-do-we-always-divide-rgb-values-by-255) and [Question 3](https://datascience.stackexchange.com/questions/29958/when-inputting-image-rgb-values-to-mlp-should-i-divide-by-255)

**Comment by rmothukuru** [comment]
Created a CL to fix this issue. Thanks!

**Comment by sachinprasadhs** [comment]
The necessary information is already present in the document as below.
```
The data must be preprocessed before training the network. If you inspect the first image
 in the training set, you will see that the pixel values fall in the range of 0 to 255:
Scale these values to a range of 0 to 1 before feeding them to the neural network model.
 To do so, divide the values by 255. It's important that the training set and the testing set
 be preprocessed in the same way:
```
For more basic information on these concepts for beginners you can refer to the crash course on ML/DL offered by google [here](https://developers.google.com/machine-learning/crash-course). Thanks.
